CMakeLists.txt

```diff
@@ -1,14 +1,39 @@
 cmake_minimum_required(VERSION 3.14)
 project(iggy-cpp-client)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # avoid warning about DOWNLOAD_EXTRACT_TIMESTAMP in CMake 3.24
 if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
   cmake_policy(SET CMP0135 NEW)
 endif()
 
+# set up library dependencies
+find_package(argh CONFIG REQUIRED)
+find_package(benchmark CONFIG REQUIRED)
+find_package(libuv CONFIG REQUIRED)
+find_package(spdlog CONFIG REQUIRED)
+
+add_library(
```

Comment:
perhaps put also headers into add_library, this way they would show nicely in the IDE (i'm using CLion)

Reply:
@hubcio my understanding is this happens automatically with vcpkg -- at least in VS Code it works. Happy to try to help fix this, did you have a particular CMake directive in mind?

Reply:
I removed cache in CLion and it looks fine 👍🏻  you can ignore this comment.